test/unit/spec/util/jobLogViewerSpec.js

```diff
@@ -35,28 +35,18 @@ define(['util/jobLogViewer', 'common/runtime'], (JobLogViewer, Runtime) => {
             });
         });
 
-        it('Should fail to start without a node', async function () {
+        it('Should fail to start without a node', async () => {
```

Comment:
This was throwing fits, and the function put in here for testing Promise rejection was added to Jasmine a while back. It's much more stable in this case.